packages/client/hmi-client/src/components/operator/tera-operator-placeholder.vue

```diff
@@ -6,56 +6,14 @@
 </template>
 
 <script setup lang="ts">
-import { WorkflowOperationTypes } from '@/types/workflow';
+import { WorkflowNode } from '@/types/workflow';
 import plants from '@/assets/svg/plants.svg';
-import calibrateJulia from '@assets/svg/operator-images/calibrate-deterministic.svg';
-import editModel from '@assets/svg/operator-images/edit-model.svg';
-import calibrateEnsembleCiemss from '@assets/svg/operator-images/calibrate-ensemble-probabilistic.svg';
-import model from '@assets/svg/operator-images/model.svg';
-import optimizeModel from '@assets/svg/operator-images/optimize-model.svg';
-import calibrateSimulateCiemss from '@assets/svg/operator-images/calibrate-simulate-probabilistic.svg';
-import simulateJulia from '@assets/svg/operator-images/simulate-deterministic.svg';
-import code from '@assets/svg/operator-images/code.svg';
-import simulateEnsembleCiemss from '@assets/svg/operator-images/simulate-ensemble-probabilistic.svg';
-import configureModel from '@assets/svg/operator-images/configure-model.svg';
-import simulateProbabilistic from '@assets/svg/operator-images/simulate-probabilistic.svg';
-import coupleModels from '@assets/svg/operator-images/couple-models.svg';
-import stratifyModel from '@assets/svg/operator-images/stratify-model.svg';
-import createModelFromCode from '@assets/svg/operator-images/create-model-from-code.svg';
-import transformDataset from '@assets/svg/operator-images/transform-dataset.svg';
-import createModelFromEquations from '@assets/svg/operator-images/create-model-from-equation.svg';
-import transformModel from '@assets/svg/operator-images/transform-model.svg';
-import dataset from '@assets/svg/operator-images/dataset.svg';
-import document from '@assets/svg/operator-images/document.svg';
 
 const props = defineProps<{
-	operationType: WorkflowOperationTypes;
+	node: WorkflowNode<any>;
```

Comment:
Why not just asking for the `imageUrl` instead of passing the whole node?

Reply:
No particular reason other than it may be handy to get access to things like description, document-url as well.